doc/classes/AnimationNodeTransition.xml

```diff
@@ -16,43 +16,49 @@
 			<return type="int" />
 			<param index="0" name="caption" type="String" />
 			<description>
+				Returns input index which corresponds to [param caption]. If not found, returns [code]-1[/code].
```

Comment:
```suggestion
				Returns the input index which corresponds to [param caption]. If not found, returns [code]-1[/code].
```

Reply:
Thanks, articles have always been my weak point, or the weak point. 🤣